src/server/api/routers/pcListings.ts

```
@@ -357,44 +357,38 @@ export const pcListingsRouter = createTRPCRouter({
-        if (hasPermission(ctx.session.user.role, Role.MODERATOR)) {
-          break
-        }
+        if (hasPermission(ctx.session.user.role, Role.MODERATOR)) break
 
         // Regular users have a time limit for editing approved listings
-        if (!pcListing.processedAt) {
-          throw AppError.badRequest('PC listing approval time not found')
-        }
+        if (!pcListing.processedAt) return ResourceError.pcListing.approvalTimeNotFound()
 
         const now = new Date()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)